doc/manual/install-domserver.rst

```diff
@@ -40,12 +40,16 @@ GNU/Linux, or one of its derivative distributions like Ubuntu::
         php-curl php-json php-xml php-zip composer ntp
 
 The following command can be used on RedHat Enterprise Linux, and related
-distributions like CentOS and Fedora::
+distributions like CentOS (with EPEL) and Fedora::
 
   sudo yum install acl zip unzip mariadb-server httpd \
         php-gd php-cli php-intl php-mbstring php-mysqlnd \
         php-xml php-zip composer ntp
 
+Due to a `bug <https://github.com/DOMjudge/domjudge/issues/862>`_ in our
```

Comment:
Nitpick: I don't think this is really a bug, but the documentation is not in line with how installation works.
```suggestion
Due to an `issue <https://github.com/DOMjudge/domjudge/issues/862>`_ in our
```